klasses/models.py

```diff
@@ -60,6 +60,7 @@ class BootcampRun(models.Model):
     run_key = models.IntegerField(unique=True, db_index=True)
     start_date = models.DateTimeField(null=True)
     end_date = models.DateTimeField(null=True)
+    bootcamp_run_id = models.CharField(null=True, unique=True, max_length=255)
```

Comment:
can we allow null values here? We have some historical bootcamps on RC and prod that won't have bootcamp_run_ids and I don't think it's worth making them up.

```suggestion
    bootcamp_run_id = models.CharField(null=True, unique=True, blank=True, max_length=255)
```

Reply:
Updated.